internal/controller/bootstrap/authentication_bootstrap_controller.go

```diff
@@ -123,6 +123,8 @@ func (r *BootstrapReconciler) writeConfigurationsToAuthenticationCR(ctx context.
 		"NONCE_ENABLED":            &authCR.Spec.Config.NONCEEnabled,
 		"PREFERRED_LOGIN":          &authCR.Spec.Config.PreferredLogin,
 		"OIDC_ISSUER_URL":          &authCR.Spec.Config.OIDCIssuerURL,
+		"AUDIT_URL":                &authCR.Spec.Config.AuditUrl,
+		"AUDIT_SECRET":             &authCR.Spec.Config.AuditSecret,
```

Comment:
These will not match the switch statement's cases; needs to be `*string`.
```suggestion
		"AUDIT_URL":                authCR.Spec.Config.AuditUrl,
		"AUDIT_SECRET":             authCR.Spec.Config.AuditSecret,
```

Reply:
fixed